// @mswjs/interceptors/lib/node/index.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});

var _chunkHAXZ5BCLjs = require('./chunk-HAXZ5BCL.js');



var _chunkLK6DILFKjs = require('./chunk-LK6DILFK.js');


var _chunk73NOP3T5js = require('./chunk-73NOP3T5.js');









var _chunkQFLITAVVjs = require('./chunk-QFLITAVV.js');

// src/utils/getCleanUrl.ts
function getCleanUrl(url, isAbsolute = true) {
  return [isAbsolute && url.origin, url.pathname].filter(Boolean).join("");
}














exports.BatchInterceptor = _chunkHAXZ5BCLjs.BatchInterceptor; exports.FetchResponse = _chunkQFLITAVVjs.FetchResponse; exports.INTERNAL_REQUEST_ID_HEADER_NAME = _chunkQFLITAVVjs.INTERNAL_REQUEST_ID_HEADER_NAME; exports.IS_PATCHED_MODULE = _chunk73NOP3T5js.IS_PATCHED_MODULE; exports.Interceptor = _chunkQFLITAVVjs.Interceptor; exports.InterceptorReadyState = _chunkQFLITAVVjs.InterceptorReadyState; exports.createRequestId = _chunkQFLITAVVjs.createRequestId; exports.decodeBuffer = _chunkLK6DILFKjs.decodeBuffer; exports.deleteGlobalSymbol = _chunkQFLITAVVjs.deleteGlobalSymbol; exports.encodeBuffer = _chunkLK6DILFKjs.encodeBuffer; exports.getCleanUrl = getCleanUrl; exports.getGlobalSymbol = _chunkQFLITAVVjs.getGlobalSymbol; exports.getRawRequest = _chunkQFLITAVVjs.getRawRequest;
